packages/usa-memorable-date/src/usa-memorable-date.twig

```diff
@@ -1,18 +1,34 @@
 <fieldset class="usa-fieldset">
-  <legend class="usa-legend">Date of birth</legend>
-  <span class="usa-hint" id="dobHint">For example, 4 28 1986</span>
+  <legend class="usa-legend">Date of Birth</legend>
+  <span class="usa-hint" id="mdHint">For example: 01 - January 19 2000</span>
   <div class="usa-memorable-date">
-    <div class="usa-form-group usa-form-group--month">
-      <label class="usa-label" for="date_of_birth_1">Month</label>
-      <input class="usa-input" aria-describedby="dobHint" id="date_of_birth_1" name="date_of_birth_1" type="text" maxlength="2" pattern="[0-9]*" inputmode="numeric" value="">
+    <div class="usa-form-group usa-form-group--month usa-form-group--select">
+      <label class="usa-label" for="date_of_birth_month">Month</label>
+      <select class="usa-select" id="date_of_birth_month" name="date_of_birth_month" aria-describedby="mdHint" >
+        <option value>- Select -</option>
+        <option value="1">01 - January</option>
+        <option value="2">02 - February</option>
+        <option value="3">03 - March</option>
+        <option value="4">04 - April</option>
+        <option value="5">05 - May</option>
+        <option value="6">06 - June</option>
+        <option value="7">07 - July</option>
+        <option value="8">08 - August</option>
+        <option value="9">09 - September</option>
+        <option value="10">10 - October</option>
+        <option value="11">11 - November</option>
+        <option value="12">12 - December</option>
```

Comment:
Values for these should be the numerical value of the month only, so I've rewritten